utilities_test.go

```diff
@@ -98,84 +98,112 @@ func TestDatasetWarp(t *testing.T) {
 	}
 }
 func TestDatasetWarpMulti(t *testing.T) {
-	ds1, _ := Open("testdata/test_1_1.tif", RasterOnly())
-	ds2, _ := Open("testdata/test_1_2.tif", RasterOnly())
-	ds3, _ := Open("testdata/test_2_1.tif", RasterOnly())
-	ds4, _ := Open("testdata/test_2_2.tif", RasterOnly())
+	ds1, _ := Create(Memory, "", 3, Byte, 5, 5)
+	ds2, _ := Create(Memory, "", 3, Byte, 5, 5)
+
+	sr, _ := NewSpatialRefFromEPSG(4326)
+	_ = ds1.SetSpatialRef(sr)
+	_ = ds2.SetSpatialRef(sr)
+
+	_ = ds1.SetGeoTransform([6]float64{45, 1, 0, 35, 0, -1})
+	_ = ds2.SetGeoTransform([6]float64{50, 1, 0, 35, 0, -1})
+
+	for _, b := range ds1.Bands() {
+		_ = b.Fill(200, 0)
+	}
+
+	for _, b := range ds2.Bands() {
+		_ = b.Fill(100, 0)
+	}
 
 	defer ds1.Close()
 	defer ds2.Close()
-	defer ds3.Close()
-	defer ds4.Close()
 
 	// Warp NewDataset with multiple input dataset
 	filePath := path.Join(os.TempDir(), "warp.tif")
-	outputDataset, err := Warp(filePath, []*Dataset{ds1, ds2, ds3, ds4}, []string{"-ts", "40", "40"}, CreationOption("TILED=YES", "BLOCKXSIZE=32", "BLOCKYSIZE=16"), GTiff)
+	outputDataset, err := Warp(filePath, []*Dataset{ds1, ds2}, []string{}, CreationOption("TILED=YES"), GTiff)
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	defer os.Remove(filePath)
 	defer outputDataset.Close()
 
-	st := outputDataset.Structure()
-	if st.SizeX != 40 || st.SizeY != 40 {
-		t.Errorf("wrong size %d,%d", st.SizeX, st.SizeY)
-	}
+	data := make([]uint8, 50)
```

Comment:
Maybe just check that the output size is 10x5 which is what is expected given the input geotransforms?